.github/workflows/test.yml

```diff
@@ -124,8 +124,8 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Build Frogbot
         run: buildscripts/build.sh
-      - name: Run GitHub Integration Tests
-        run: go test -v -p 1 github_test.go integrationutils.go main.go commands.go -race -timeout 30m -cover
+      - name: Run Integration Tests
+        run: go test -v github_test.go azure_test.go integrationutils.go main.go commands.go -race -timeout 30m -cover
```

Comment:
Let's separate the github_test.go and azure_test.go to different test suites. I'd even go further and put them under `azureTest.yml` and `githubTest.yml`.
Make sure to include the `FROGBOT_TESTS_AZURE_TOKEN` token in the Azure suite.